python/perspective/package.json

```diff
@@ -3,7 +3,8 @@
   "name": "perspective-python.node",
   "version": "0.4.1",
   "dependencies": {
-    "@finos/perspective": "^0.4.1"
+    "@finos/perspective": "^0.4.1",
+    "zerorpc": "^0.9.8"
```

Comment:
@texodus please confirm

Reply:
At this point I see honestly zero reason to maintain this feature - `perspective-python` is better in every way.